app/models/like.rb

```diff
@@ -1,2 +1,13 @@
 class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
Kindly fix the belong here to author, class_name: user,   the same way you did it in other files